modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -744,7 +744,7 @@ public String toApiName(String name) {
             return "DefaultApi";
         }
         name = sanitizeName(name);
-        return camelize(name) + "Api";
+        return camelize(apiNamePrefix + "_" + name + "_" + apiNameSuffix);
```

Comment:
Would it be better to instead modify `DefaultCodegen.java` to have `sanitizeName()` and then delete this overidden method?

Reply:
I don't know the implications on the other generators, that already exist.

Reply:
It would sanitize the name, but otherwise I think would behave the same way as it currently does.

@wing328 any thoughts on this?